deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4938,12 +4997,26 @@ NAN_METHOD(WebGLRenderingContext::GetSupportedExtensions) {
   // glGetIntegerv(GL_NUM_EXTENSIONS, &numExtensions);
 
   int numExtensions = sizeof(webglExtensions)/sizeof(webglExtensions[0]);
-  Local<Array> result = Nan::New<Array>(numExtensions);
+  Local<Array> result = Nan::New<Array>(0);
   for (GLint i = 0; i < numExtensions; i++) {
-    // char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
-    result->Set(i, JS_STR(webglExtensions[i]));
+    char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
+    if(extension == webglExtensions[i]){
+      string currentExtension = string(webglExtensions[i]);
+      bool hasWebGL = currentExtension.find("WEBGL");
```

Comment:
This is probably not a reliable check -- we should instead manually chop the array above to make our intent clear. It would also be faster.